src/types/Wallet/Wallet.d.ts

```diff
@@ -59,5 +59,10 @@ export declare namespace Wallet {
         privateKey?: PrivateKey | string;
         HDPrivateKey?: HDPrivateKey | string;
         HDPublicKey?: HDPublicKey | string;
+        debugOptions?: IWalletDebugOptions;
```

Comment:
What is the point to introduce this namespace?

Reply:
Why not?

Reply:
I was trying to follow this issue https://github.com/dashevo/wallet-lib/issues/216

Reply:
debugOptions sounds confusing honestly. What if we rename this namespace to `unsafe`? Or rename the option to `unsafeSkipSynchronizationBeforeHeight`? @antouhou